vhost-user-backend/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 ### Added
 
 ### Changed
+- [[#262]](https://github.com/rust-vmm/vhost/pull/262) Update virtio-queue
+dependency to 0.14.0 and vm-memory to 0.16.0; move rust-vmm crates to workspace.
```

Comment:
entries below tended to ident the wrapped line to match the part after `- `.

Reply:
Noted, and I suggest we use `mdformat` to take care of formatting :)